schema/newschma.yaml

```diff
@@ -0,0 +1,32 @@
+root:
+  parts: # We dropped the SEQUENCE information from the excel sheet
+   name: string
+   role: enum
+   circular: boolean # isn't this implicit from role ?
+   partDescription: string
+   application: 
```

Comment:
application is defining a Device

Reply:
Yup, based on how the Functional Synthetic Biology Paper is going, this would be describing a device. However, I'd like to retain the `application context` terminology to ensure that this section becomes the melting point for all the data coming from: `Device` + `Characterization` + `whatever`. The core of what we want to describe is reproducible designs.